java/Cargo.toml

```diff
@@ -18,6 +18,7 @@ tracing-subscriber = "0.3.16"
 jni = "0.21.1"
 log = "0.4.20"
 bytes = { version = "1.6.0" }
+serde = "1.0.219"
```

Comment:
why?

Reply:
Very weird. CI can build java on `main`. Is it something alpine related?

Reply:
Yes, I don't remember what exactly atm, I can track it down if there's concern about this change